plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/InviteCommand.java

```diff
@@ -68,4 +117,41 @@ public void execute(@CanManage User deputy, Guild guild, String[] args) {
         invitedUser.sendMessage(formatter.format(this.messages.inviteToInvited));
     }
 
+    public static final class InviteCommandCompleter implements Completer {
+
+        private final PluginConfiguration configuration;
+        private final UserManager userManager;
+
+        public InviteCommandCompleter(PluginConfiguration configuration, UserManager userManager) {
+            this.configuration = configuration;
+            this.userManager = userManager;
+        }
+
+
+        @Override
+        public String getName() {
+            return "invite-command";
+        }
+
+        @Override
+        public List<String> apply(Context context, String prefix, Integer limit) {
+            String[] args = context.getArguments();
+
+            if (args.length == 1) {
+
+                List<String> toReturn = Bukkit.getOnlinePlayers().stream()
+                        .map(it -> this.userManager.findByPlayer(it).get())
+                        .filter(it -> !it.hasGuild())
+                        .filter(it -> !it.isVanished())
+                        .map(User::getName)
+                        .collect(Collectors.toList());
+                toReturn.add(this.configuration.inviteCommandAllArgument);
+
+                return toReturn;
+            }
+            else {
+                return Lists.newArrayList("");
+            }
```

Comment:
Możesz sobie to dać o poziom w dół.
```suggestion
        if (args.length != 1) {
            return Lists.newArrayList("");
        }

        List<String> toReturn = Bukkit.getOnlinePlayers().stream()
                .map(it -> this.userManager.findByPlayer(it).get())
                .filter(it -> !it.hasGuild())
                .filter(it -> !it.isVanished())
                .map(User::getName)
                .collect(Collectors.toList());
        toReturn.add(this.configuration.inviteCommandAllArgument);

        return toReturn;
```

Reply:
`.map(it -> this.userManager.findByPlayer(it).get())` to trochę unsafe weź PandaStream i mapOpt